CONSTITUTION.md

```diff
@@ -162,7 +163,7 @@ In the absence of both the President and the Vice-President, the Committee membe
 
 ### 7.2. Annual General Meetings
 
-- 7.2.1. The Annual General Meeting shall be held in the month of October each year.
+- 7.2.1. The Annual General Meeting shall be held in either September or October of each year.
```

Comment:
Why?

Reply:
(Updated rationale too) Gives more options to pick a date, given the "First Tuesday in November" deadline for re-affiliation clubs have. Activate allows for a wider AGM season in current policy.